tooling/nargo_cli/Cargo.toml

```diff
@@ -85,6 +86,7 @@ sha2.workspace = true
 sha3.workspace = true
 iai = "0.1.1"
 test-binary = "3.0.2"
+light-poseidon = { git = "https://github.com/Lightprotocol/light-poseidon.git", tag = "v0.2.0" }
```

Comment:
Looks like it's been published to crates.io (https://crates.io/crates/light-poseidon) so we don't need the git dependency?

Reply:
Ah cool! 🦅 👁️